Userland/Libraries/LibGfx/AntiAliasingPainter.h

```diff
@@ -77,11 +77,11 @@ class AntiAliasingPainter {
 
     void draw_dotted_line(IntPoint, IntPoint, Gfx::Color, int thickness);
 
-    enum class AntiAliasPolicy {
-        OnlyEnds,
-        Full,
+    enum class PathCompatibility {
+        Disable,
+        Enable,
```

Comment:
I have no idea what this means. Can we call it something else that makes it clear what the purpose is?

Reply:
Sure, I can't think of an elegant name, but it's really `FixmeEnableHacksForBetterPathPainting::Yes/No`

Reply:
:yakkiss: